src/cirrus/github_tools.py

```diff
@@ -269,7 +269,25 @@ def tag_release(self, tag, master='master', push=True):
             raise RuntimeError(msg)
         self.repo.create_tag(tag)
         if push:
-            self.repo.remotes.origin.push(self.repo.head, tags=True)
+            count = 0
+            error_flag = None
+            while count < attempts:
+                error_flag = None
+                try:
+                    self.repo.remotes.origin.push(self.repo.head, tags=True)
+                    break
+                except Exception as ex:
+                    msg = "Error pushing tags: {}".format(ex)
+                    LOGGER.info(msg)
+                    error_flag = ex
+                    time.sleep(cooloff)
```

Comment:
If this is a cool off, don't you want to scale with attempts too? 2s, 4s, 8s, that sort of thing?